### ISSUE TITLE
SplitRows and SplitColumns without border

### ISSUE THREAD

**Issue by agbarbosa** [goblinfactory/konsole#71]
Please add to the LineThickness ENUM the "none" option. Currently we have only the Single and Double ones.

**Comment by goblinfactory** [comment]
absolutely... oversight on my part, this should have been in from the start.
will add to next release as priority.

**Comment by phamduylong** [comment]
Doesn't look like it's implemented yet. @goblinfactory Would you mind if I give this a go?

**Comment by goblinfactory** [comment]
Hi @phamduylong : Thank you for offering to work on one of the features, unfortunately Goblinfactory Konsole has too many branches exploring options for version 7 and 8 that already rolls up changes that makes this feature redundant. Until I bring out version 7 and 8 the project won't be a good one to volunteer on. I wish you the best of luck finding an open source project to work on. 
regards, 
Alan

**Comment by phamduylong** [comment]
Thanks for the heads up Alan. I'll keep an eye out.
